revise content of requesting access page (#443)
ASFHyP3/hyp3-docs: docs/using/requesting_access.md

# Requesting Access

You must request access before submitting jobs for On Demand processing using the HyP3 platform. HyP3 accounts are limited to one per person.

[Request Access](#){ .md-button .md-button--primary .middle }

You will need an [Earthdata Login](https://urs.earthdata.nasa.gov/ 'https://urs.earthdata.nasa.gov/' ){target=_blank} (EDL) account to complete this request. You can [register here](https://urs.earthdata.nasa.gov/users/new 'https://urs.earthdata.nasa.gov/users/new' ){target=_blank} if you do not already have an account. 

You will receive an email with your approval status within two business days. If you have not already registered for HyP3 access, you can expect to be approved.

Once your access request has been approved, you can submit jobs for On Demand processing using your EDL credentials. *You only need to complete this approval process once.* 

> 

Reviewer: I think there still is merit to including some clarification on the grandfathering process. Maybe moving it down to this location? 
```suggestion

!!! tip "Existing HyP3 users do not need to request access" 

    Existing On Demand users will *not* need to undergo this process unless they have been identified as having multiple HyP3 accounts. If you are an existing user who is no longer able to process On Demand jobs, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.
```

# Requesting Access

You must request access before submitting jobs for On Demand processing using the HyP3 platform. HyP3 accounts are limited to one per person.

[Request Access](#){ .md-button .md-button--primary .middle }

You will need an [Earthdata Login](https://urs.earthdata.nasa.gov/ 'https://urs.earthdata.nasa.gov/' ){target=_blank} (EDL) account to complete this request. You can [register here](https://urs.earthdata.nasa.gov/users/new 'https://urs.earthdata.nasa.gov/users/new' ){target=_blank} if you do not already have an account. 

You will receive an email with your approval status within two business days. If you have not already registered for HyP3 access, you can expect to be approved.

Once your access request has been approved, you can submit jobs for On Demand processing using your EDL credentials. *You only need to complete this approval process once.* 

!!! tip "Existing HyP3 users do not need to request access" 

>     Existing On Demand users will *not* need to undergo this process unless they have been identified as having multiple HyP3 accounts. If you are an existing user who is no longer able to process On Demand jobs, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.

Reviewer 2: ```suggestion
    Existing On Demand users (i.e., users who signed up before April 2024) will *not* need to undergo this process. If you are no longer able to process On Demand jobs and you believe you should still have access, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.
```

>     Existing On Demand users (i.e., users who signed up before April 2024) will *not* need to undergo this process. If you are no longer able to process On Demand jobs and you believe you should still have access, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.

Reviewer 2: ```suggestion
    Existing On Demand users (i.e. users who signed up before May 2024) will *not* need to undergo this process. If you are no longer able to process On Demand jobs and you believe you should still have access, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.
```
